api/src/handlers/helpers/getTemplateDataFromInputs.ts

```diff
@@ -0,0 +1,43 @@
+import { InputFields } from "./getAllInputsFromForm";
+import { AffirmationFieldList, fieldLists } from "./fieldLists";
+import { YesNoMap } from "./fieldLists/common";
+import { CNIFieldList } from "./fieldLists/cni";
+import { Errors } from "./buildEmailData";
+
+export type AffirmationTemplateData = Record<AffirmationFieldList, string>;
+export type CNITemplateData = Record<CNIFieldList, string>;
+
+export interface TemplateData {
+  templateVars: Partial<AffirmationTemplateData | CNITemplateData>;
+  uploads: {
+    [key: string]: string;
+  };
+}
+
+export function getTemplateDataFromInputs(
```

Comment:
I'm not using this for now. I'm just accepting whatever the form json is. we're not doing any fancy logic on the data, so iterating through questions is fine.. famous last words